commcare_connect/templates/tailwind/components/pagination.html

```diff
@@ -0,0 +1,18 @@
+<div class="flex items-center gap-2 max-w-fit text-brand-deep-purple text-sm">
```

Comment:
how is this used? basically how would we make a paginated table?

Reply:
I have updated the code. This is now removed with pagination now going into base_table itself